tools/generators/normalize-package-dependencies/schema.ts

```diff
@@ -0,0 +1,14 @@
+export interface NormalizePackageDependenciesGeneratorSchema {
+  /**
+   * Filter flag. Use to apply generator execution only on projects that contain provided `projectType` within their `project.json#projectType`.
+   */
+  projectType?: 'application' | 'library' | 'any';
```

Comment:
do we need such generic handling of project types? seems like someone could do a great evil and set all library dependencies to be `*`.

Reply:
that's very good concern - but on the ci we will run `verify` with projectType=application. so if someone did that by accident , CI would fail.